js/utils/parsers/tag.js

```diff
@@ -5,8 +5,7 @@ const SHOW_LESS_SELECTOR = 'paper-button.ytd-expander:nth-child(3)';
 const TAG_SELECTOR = '#collapsible ytd-metadata-row-renderer:last-child #content';
 const YOUTUBE_TAG_REGEX = /"([^()]*)(.*)?" by (.*) \(.*•?.*\)/;
 
-const showMoreButton = async () => 
-  await awaitElement(SHOW_MORE_SELECTOR);
```

Comment:
This `await` wrapped the result in an extra promise. We can just use the inner promise to cater for future chaining.

Reply:
You might want to look up whether I was right to remove the `async`, I'm fairly sure that this is written fine as a standard function (just return a promise).

Reply:
It does, thanks for spotting that.